requirements.txt

```diff
@@ -18,17 +18,17 @@ fprime-fpl-convert-xml==1.0.3
 fprime-fpl-extract-xml==1.0.3
 fprime-fpl-layout==1.0.3
 fprime-fpl-write-pic==1.0.3
-fprime-fpp-check==2.1.0a6
-fprime-fpp-depend==2.1.0a6
-fprime-fpp-filenames==2.1.0a6
-fprime-fpp-format==2.1.0a6
-fprime-fpp-from-xml==2.1.0a6
-fprime-fpp-locate-defs==2.1.0a6
-fprime-fpp-locate-uses==2.1.0a6
-fprime-fpp-syntax==2.1.0a6
-fprime-fpp-to-cpp==2.1.0a6
-fprime-fpp-to-json==2.1.0a6
-fprime-fpp-to-xml==2.1.0a6
+fprime-fpp-check==2.1.0a7
+fprime-fpp-depend==2.1.0a7
+fprime-fpp-filenames==2.1.0a7
+fprime-fpp-format==2.1.0a7
+fprime-fpp-from-xml==2.1.0a7
+fprime-fpp-locate-defs==2.1.0a7
+fprime-fpp-locate-uses==2.1.0a7
+fprime-fpp-syntax==2.1.0a7
+fprime-fpp-to-cpp==2.1.0a7
+fprime-fpp-to-json==2.1.0a7
+fprime-fpp-to-xml==2.1.0a7
```

Comment:
We should add a line for 
```
fprime-fpp-to-dict==2.1.0a7
```

It has been successfully released: https://pypi.org/project/fprime-fpp-to-dict/

Reply:
Yes please do!

Reply:
Done!